assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -108,9 +104,9 @@ describe("DiversionPage", () => {
     )
 
     const intersection = "Avenue 1 & Street 2"
-    jest.mocked(useNearestIntersectionFetchResult).mockReturnValue({
-      ok: intersection,
-    })
+    jest
+      .mocked(fetchNearestIntersection)
+      .mockReturnValue(Promise.resolve(intersection))
```

Comment:
nit: can we prefer `mockResolvedValue` here, and on `L524`?

Reply:
Oh I can try it - I thought based on our pairing session that this was more predictable.